content/millennium/dstu2/scheduling/appointment.md

```diff
@@ -102,7 +103,7 @@ List an individual Appointment by its id:
 
 #### Request
 
-    GET https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/MedicationStatement/22298077
+    GET https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Appointment/22298077
```

Comment:
This request fails - this should be the request the user should be able to use to get the resulting example response. It does need to be real/something they can try in sandbox